src/index.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { Menu } from "./components/Menu";
+import { OrderSummary } from "./components/Order";
+import "./styles/main.scss";
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      orderItems: [],
+      orderTotal: 0.0
+    };
+    this.addToOrder = this.addToOrder.bind(this);
+    this.removeFromOrder = this.removeFromOrder.bind(this);
+  }
+
+  addToOrder(name, price, bread, isHot) {
+    const orderItems = this.state.orderItems;
+    const orderTotal = this.state.orderTotal;
+
+    orderItems.push({
+      name: name,
+      price: price,
+      bread: bread,
+      isHot: isHot
+    });
+
+    this.setState({
+      orderItems: orderItems,
+      orderTotal: orderTotal + price
+    });
+  }
+
+  removeFromOrder(i, price) {
+    const orderItems = this.state.orderItems;
+    const orderTotal = this.state.orderTotal;
+
+    orderItems.splice(i, 1);
+
+    this.setState({
+      orderItems: orderItems,
+      orderTotal: orderItems.length === 0 ? 0 : orderTotal - price
```

Comment:
Can move this calculation out of this statement then you can benefit from the shorthand 👍